numba/cpython/cmathimpl.py

```diff
@@ -151,6 +151,7 @@ def exp_impl(x, y, x_is_finite, y_is_finite):
             s = math.sin(y)
             return complex(r * c, r * s)
         else:
+            r = 0
```

Comment:
This its technically `complex(-0, 0)` in this case i.e. `cmath.exp(-cmath.inf + 2j)`, i.e. signed zero, but that's most probably an implementation detail?

Reply:
it's what the python implementation would do.

Reply:
it's already tested in test_complex